efficientdet/keras/efficientdet_arch_keras.py

```diff
@@ -210,3 +217,468 @@ def get_config(self):
     }
     base_config = super(ResampleFeatureMap, self).get_config()
     return dict(list(base_config.items()) + list(config.items()))
+
+
+class ClassNet(tf.keras.layers.Layer):
+  """Class prediction network.
+
+  Args:
+  num_classes: number of classes
+  num_anchors: number of anchors (usually 9)
+  num_filters: number of filters for "intermediate" layers
+  min_level: minimum level for features (usually 3)
+  max_level: maximum level for features (usually 7)
+  is_training_bn: True if we train the BatchNorm
+  act_type: String of the activation used
+  data_format: channel_first if [B, C, W, H] format, 'channels_last' if [B, W, H, C]
+  box_class_repeats: number of "intermediate" layers
+  separable_conv: True to use separable_conv instead of conv2D
+  survival_prob: if a value is set then drop connect will be used with this probability
+  use_tpu: True to use TPU compatible functions
+
+  """
+  def __init__(self,
+               num_classes=80,
```

Comment:
num_classes = 80?

Isn't the default in this codebase 90?

Reply:
Yes, i'm just used to 80, but as 90 is the default config one it shouldn't have any influence, I'll change it and it'll come within next commit

Reply:
Hi please use 90 classes to avoid confusion. COCO 2012 is not simply using class 0-80, see this: https://tech.amikelive.com/node-718/what-object-categories-labels-are-in-coco-dataset/. Thank you!